modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/lizard.dm

```diff
@@ -1,6 +1,7 @@
 /datum/species/lizard
 	mutant_bodyparts = list()
 	species_traits = list(MUTCOLORS,EYECOLOR,LIPS,HAS_FLESH,HAS_BONE,HAIR,FACEHAIR)
+	inherent_traits = list(TRAIT_CAN_USE_FLIGHT_POTION)
```

Comment:
I'm pretty sure that it already works on lizards, and if you do this, it'll overwrite some stuff we really don't want to override.
(so remove it)
```suggestion
```